### PR TITLE
handle illegal argument exception

### PR DESCRIPTION
**Pull request by evgeniilezhnin**

### CODE REVIEW

File: backend/common/src/main/java/ai/verta/modeldb/common/CommonUtils.java [hunk 1]
```diff
@@ -130,6 +130,15 @@ public static <T extends GeneratedMessageV3> StatusRuntimeException logError(
...
                 .build();
+      } else if (e instanceof IllegalArgumentException) {
+        var illegalArgumentException = (IllegalArgumentException) e;
+        logBasedOnTheErrorCode(
+            isClientError(Code.INVALID_ARGUMENT_VALUE), illegalArgumentException);
+        status =
+            Status.newBuilder()
+                .setCode(Code.INVALID_ARGUMENT_VALUE)
+                .setMessage(illegalArgumentException.getMessage())
+                .build();
```
Comment (anandjakhaniya) [inline backend/common/src/main/java/ai/verta/modeldb/common/CommonUtils.java:141]: illegalArgumentException will thrown due to some internal mistakes not due to client's invalid argument i think. NO?

Comment (evgeniilezhnin) [reply backend/common/src/main/java/ai/verta/modeldb/common/CommonUtils.java:141]: Looks like it is used for client argument validation in library code and also in generated grpc code. Also we use it in our code for agrument validation. As an alternative we can remove all usages of this class from our code. Is it really needed?

Comment (anandjakhaniya) [reply backend/common/src/main/java/ai/verta/modeldb/common/CommonUtils.java:141]: In the modeldb and registry backend i am sure i do not used it anywhere for client parameter validation.

Comment (evgeniilezhnin) [reply backend/common/src/main/java/ai/verta/modeldb/common/CommonUtils.java:141]: For example: https://github.com/VertaAI/modeldb/blob/553e74086a34ce65254b8f0e89dea1c16b1a902d/backend/src/main/java/ai/verta/modeldb/project/FutureProjectDAO.java#L1735

Comment (evgeniilezhnin) [reply backend/common/src/main/java/ai/verta/modeldb/common/CommonUtils.java:141]: That doesn't look like internal error code, more like invalid argument

Comment (anandjakhaniya) [reply backend/common/src/main/java/ai/verta/modeldb/common/CommonUtils.java:141]: Yeah got it. Ok, then we can add it.

**Review by anandjakhaniya — APPROVED** [review]

**Pull request merged**